### PR TITLE
Setup social auth logins and dynamic user profile paths

### PR DESCRIPTION
**Pull request by alternadiva**

### CODE REVIEW

**Review by minju25kim — APPROVED** [review]

File: pages/profiles/[id].js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import supabase from "../../utils/supabaseClient";
+
+function Profile(props) {
+  return (
+    <section>
+      <h1>Profile</h1>
```
Comment (AsmahanM) [inline pages/profiles/[id].js:6]: Nice work here! I can see the profile page coming together.

**Review by AsmahanM — APPROVED** [review]

File: pages/profiles/[id].js (hunk 1 above)
Comment (sonianb) [inline pages/profiles/[id].js:6]: Maybe that would be an h2 tag? But we can change that later!

**Comment by sonianb** [comment]
[Based on signup data display user information on user profile.](https://trello.com/c/vCDot6Om/58-based-on-signup-data-display-user-information-on-user-profile)

**Comment by sonianb** [comment]
[Configure social authentication for Facebook and Google.](https://trello.com/c/PNgBum9f/27-configure-social-authentication-for-facebook-and-google)

**Pull request merged**